src/components/POLine/Location/LocationView.js

```diff
@@ -117,7 +130,7 @@ const LocationView = ({
       {
         lineLocations.map((location, i) => (
           <Location
-            key={location.id || i}  // i is required when new row of Location is added by User
+            key={location.holdingId || i}  // i is required when new row of Location is added by User
```

Comment:
why?

Reply:
I have checked the obeject dosn't provide `id` it provides `holdingId` instead

Reply:
https://s3.amazonaws.com/foliodocs/api/mod-inventory-storage/r/location.html#locations_get

Reply:
[LocationsView](https://github.com/folio-org/ui-orders/blob/master/src/components/POLine/Location/LocationView.js#L102) <- [POLineView](https://github.com/folio-org/ui-orders/blob/master/src/components/POLine/POLineView.js#L465) <- [POLine](https://github.com/folio-org/ui-orders/blob/master/src/components/POLine/POLine.js#L64)

Reply:
Oh, I look at another locations...

Reply:
Yes, you're right. But in that case it could be either `holdingId` or `locationId`